blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/BlueTestResultContainerImpl.java

```diff
@@ -147,4 +178,29 @@ public boolean apply(@Nullable BlueTestResult input) {
             return input != null && input.getTestState().equals(state);
         }
     }
+
+    static class AgePredicate implements Predicate<BlueTestResult> {
```

Comment:
This doesn't appear to bind to anything in the parent class, that I can see (it's hard to be sure using GH instead of Idea, so ignore me if I'm wrong) but if it can be moved to file scope rather than nested it's probably nicer to do so. It surely makes little difference in this case, but no need to bloat classes if we don't need to.

Reply:
Agreed, but also super lazy, and since they were there already, Imma going to leave them.